requirements/requirements.txt

```diff
@@ -1,5 +1,5 @@
-ovos-utils < 0.2.0, >=0.0.37
-ovos-bus-client < 0.2.0, >=0.0.8
+ovos-utils < 0.2.0, >=0.0.38
+ovos-bus-client < 0.2.0, >=0.0.9a3
```

Comment:
Needs a stable release of ovos-bus-client and version bump here before stable release

Reply:
blocked by  https://github.com/OpenVoiceOS/ovos-bus-client/pull/107